### PR TITLE
Feature/JC19-269 film deletion

### PR DESCRIPTION
**Pull request by artem-sotnichenko-saritasa**

### CODE REVIEW

File: vanilla/src/film/index.html [hunk 1]
```diff
@@ -8,8 +8,16 @@
     <title>film</title>
   </head>
   <body>
-    <main class='film-page'>
+    <main class='film'>
+      <div class='film__header'>
+        <h1 class="film__title"></h1>
+        <div class='film__buttons'>
+          <button class='film__change is-secondary'>Change</button>
```
Comment (golovachev-yaroslav) [inline vanilla/src/film/index.html]: It would be better to add `type="button"`

**Review by golovachev-yaroslav — APPROVED** [review]

File: vanilla/src/film/display-film.ts [hunk 2]
```diff
@@ -5,11 +5,11 @@ import { ConnectedFilm } from '../entities/connected-film/types';
  * Create film element.
  * @param film Film data.
  */
-function createFilmEl(film: ConnectedFilm): HTMLDivElement {
-  const node = document.createElement('div');
-  node.classList.add('film');
-  node.innerHTML = `
-<h1 class="film__title">${film.title}</h1>
+function createFilmInfoEl(film: ConnectedFilm): HTMLDivElement {
+  const infoEl = document.createElement('div');
```
Comment (panov-aleksey-saritasa) [inline vanilla/src/film/display-film.ts:9]: Please, [avoid contractions](https://frontendwiki.saritasa.rocks/pages/development/principles/naming.html#avoid-contractions)

File: vanilla/src/index.html [hunk 3]
```diff
@@ -8,32 +8,42 @@
...
-            <option value='fields.director'>director</option>
-            <option value='fields.release_date'>release date</option>
+      <div class="films__controls">
+        <div class="films__sort films__sort--field sort">
+          <label class="sort__label" for="sort-field">sort field</label>
+          <select class="sort__select" name="sort-field" id="sort-field">
+            <option value="fields.title">title</option>
+            <option value="fields.producer">producer</option>
+            <option value="fields.director">director</option>
+            <option value="fields.release_date">release date</option>
```
Comment (panov-aleksey-saritasa) [inline vanilla/src/index.html:20]: Is `fields.` something related to `Firebase`? I think UI should not know about such things.

**Review by timofei-maximov-saritasa — APPROVED** [review]

**Review by golovachev-yaroslav — APPROVED** [review]

File: vanilla/src/modal/index.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
+interface ModalOptions {
+
+  /** Modal title. */
+  title: string;
```
Comment (pavel-romanov8) [inline vanilla/src/modal/index.ts:4]: I don't see that you are mutating those fields in the function below. Let's make them read-only to be more explicit

**Review by gevorg-gevorgyan-saritasa — APPROVED** [review]

File: vanilla/src/auth/login/index.html [hunk 5]
```diff
@@ -1,34 +1,38 @@
...
-  <script type="module" src="./index.ts"></script>
-</body>
+  <body>
+    <form class="auth-form">
+      <h1>log in</h1>
+      <div class="auth-form__error"></div>
+      <div class="auth-form__field">
+        <label for="email">email: </label>
+        <input class="auth-form__email" id="email" type="email" required />
+      </div>
```
Comment (pavel-romanov8) [inline vanilla/src/auth/login/index.html:19]: Can we reduce the number of div wrappers like this:
```
<label>
  <input>
</label>
```
?
That way you can even omit `for` and `id` attributes.

**Review by pavel-romanov8 — CHANGES_REQUESTED** [review]

**Review by elena-dragunova — APPROVED** [review]

**Review by golovachev-yaroslav — APPROVED** [review]

**Review by dbitkovski — APPROVED** [review]

**Pull request merged**